src/Kunstmaan/AdminBundle/Resources/config/services.yml

```diff
@@ -174,7 +174,7 @@ services:
             - { name: cache.pool }
 
     kunstmaan_admin.form.type.color:
```

Comment:
Both the services can be deleted as the symfony form types are registered by symfony themself. We don't need the extra services anymore.

Reply:
I'll strip them completely out then! Thanks  again!